config/config.go

```diff
@@ -54,22 +53,6 @@ type AttrConfig struct {
 	DeleteProcessedData *bool             `json:"delete_processed_data"`
 }
 
-// NewAttrConfig creates a SLAM config from a service config.
-func NewAttrConfig(cfg config.Service) (*AttrConfig, error) {
```

Comment:
I assume we don't need to remove this from the rdk repo because we're going to remove all of slam_copy in https://viam.atlassian.net/browse/RSDK-2508. Is that correct?

Reply:
Correct. This function is not used anywhere.